File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/ExperimentAssignmentVO.kt
package com.sdkwork.app

data class ExperimentAssignmentVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val experimentKey: String? = null,
    val variantKey: String? = null,
    val variantName: String? = null,
    val assignmentReason: String? = null,
    val assignmentParams: Map<String, Any>? = null
)

File: sdkwork-app-sdk-typescript/src/types/experiment-assignment-vo.ts
/** 实验分组响应 */
export interface ExperimentAssignmentVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 实验标识 */
  experimentKey?: string;
  /** 变体标识 */
  variantKey?: string;
  /** 变体名称 */
  variantName?: string;
  /** 分配原因 */
  assignmentReason?: string;
  /** 分配参数 */
  assignmentParams?: Record<string, unknown>;
}
